docs/path-resolution.rst

```diff
@@ -470,6 +472,10 @@ importing files from the following directories:
     The case must exactly match the one used in your imports.
     For example ``--allow-paths tokens`` will not match ``import "Tokens/IERC20.sol"``.
 
+.. note::
+
+    Allowed paths is incompatible with the ``--no-import-callback`` option.
```

Comment:
```suggestion
    Specifying allowed paths is incompatible with the ``--no-import-callback`` option.
    The protection against loading files from unexpected locations is a feature
    of the Host Filesystem Loader, which the option disables.
```

But actually, does it even make sense to mention it? It's just a small detail that you'll find out right away if you try to use them together. I think it needs no explanation. Definitely nothing that stands out and draws attention, like a note.

Reply:
Sure, I'll remove it. I guess it's even implicitly implied in the above section.